canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.jsx

```diff
@@ -61,6 +61,7 @@ class ToggleControl extends React.Component {
 		const className = classNames("properties-toggle", { "hide": this.props.state === STATES.HIDDEN }, this.props.messageInfo ? this.props.messageInfo.type : null);
 		return (
 			<div className={className} data-id={ControlUtils.getDataId(this.props.propertyId)}>
+				{this.props.control.label.text}
```

Comment:
This isn't the correct way to do this.  The toggle control has labelText as a property.  That should be used instead.